### PR TITLE
Fixing the tier2-cluster-elasticity suite

### PR DESCRIPTION
**Pull request by tintumathew10**
# Description

Fixing the tier2-cluster-elasticity suite.

logs - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-PWIBJ0/

### CODE REVIEW

File: suites/reef/cephadm/tier2-cluster-elasticity.yaml [hunk 1]
```diff
@@ -603,12 +570,12 @@ tests:
         pool_type: 'normal'
       desc: run rados bench for 360 - normal profile
 
-  - test:
```
Comment (vamahaja) [inline suites/reef/cephadm/tier2-cluster-elasticity.yaml:606]: why this test case is skipped ?

File: suites/reef/cephadm/tier2-cluster-elasticity.yaml [hunk 2]
```diff
@@ -724,50 +688,27 @@ tests:
...
-        args:
-          placement:
-            nodes:
-              - node10
-            limit: 2
-            sep: ";"
-      destroy-cluster: false
-      abort-on-fail: true
 
-  - test:
```
Comment (vamahaja) [inline suites/reef/cephadm/tier2-cluster-elasticity.yaml:751]: why this test case is skipped ?

File: suites/reef/cephadm/tier2-cluster-elasticity.yaml [hunk 3]
```diff
@@ -785,12 +726,12 @@ tests:
         pool_type: 'normal'
       desc: run rados bench for 360 - normal profile
 
-  - test:
```
Comment (vamahaja) [inline suites/reef/cephadm/tier2-cluster-elasticity.yaml:788]: why this test case is skipped ?

**Review by vamahaja — CHANGES_REQUESTED** [review]

File: suites/reef/cephadm/tier2-cluster-elasticity.yaml (hunk 1 above)
Comment (tintumathew10) [reply suites/reef/cephadm/tier2-cluster-elasticity.yaml:606]: rbd-io was throwing socket timeout at times. so commented it.

File: suites/reef/cephadm/tier2-cluster-elasticity.yaml (hunk 2 above)
Comment (tintumathew10) [reply suites/reef/cephadm/tier2-cluster-elasticity.yaml:751]: rbd-io was throwing socket timeout at times. so commented it

**Comment by tintumathew10** [comment]
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-HL31MZ/

File: suites/reef/cephadm/tier2-cluster-elasticity.yaml [hunk 4]
```diff
@@ -603,12 +570,12 @@ tests:
...
-      name: rbd-io
-      module: rbd_faster_exports.py
-      config:
-        io-total: 100M
-      desc: Perform export during read/write,resizing,flattening,lock operations
+  # - test:
```
Comment (vamahaja) [inline suites/reef/cephadm/tier2-cluster-elasticity.yaml:573]: Add note with reason to skip this tc

File: suites/reef/cephadm/tier2-cluster-elasticity.yaml [hunk 5]
```diff
@@ -785,12 +726,12 @@ tests:
...
```
Comment (vamahaja) [inline suites/reef/cephadm/tier2-cluster-elasticity.yaml:729]: Add note with reason to skip this tc

**Review by vamahaja — CHANGES_REQUESTED** [review]

File: suites/reef/cephadm/tier2-cluster-elasticity.yaml (hunk 4 above)
Comment (tintumathew10) [reply suites/reef/cephadm/tier2-cluster-elasticity.yaml:574]: done

File: suites/reef/cephadm/tier2-cluster-elasticity.yaml (hunk 5 above)
Comment (tintumathew10) [reply suites/reef/cephadm/tier2-cluster-elasticity.yaml:731]: done

**Pull request merged**